modules/javafx.fxml/src/test/java/test/javafx/fxml/RT_18933Test.java

```diff
@@ -28,12 +28,15 @@
 import java.io.IOException;
 import javafx.fxml.FXMLLoader;
 import javafx.fxml.LoadException;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class RT_18933Test {
-    @Test(expected=LoadException.class)
+    @Test
     public void testDefaultListProperty() throws IOException {
-        FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("rt_18933.fxml"));
-        fxmlLoader.load();
+            assertThrows(LoadException.class, () -> { FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("rt_18933.fxml"));
```

Comment:
formatting looks off here as well, especially `assertThrows` is indented too much